docs/ref/forms/api.txt

```diff
@@ -830,107 +830,203 @@ The field-specific output honors the form object's ``auto_id`` setting::
+
+    The name that will be used in the widget's HTML ``name`` attribute. It takes
+    the form :attr:`~django.forms.Form.prefix` into account.
 
 .. attribute:: BoundField.id_for_label
 
-Use this property to render the ID of this field. For example, if you are
-manually constructing a ``<label>`` in your template (despite the fact that
-:meth:`~BoundField.label_tag` will do this for you):
+    Use this property to render the ID of this field. For example, if you are
+    manually constructing a ``<label>`` in your template (despite the fact that
+    :meth:`~BoundField.label_tag` will do this for you):
+
+    .. code-block:: html+django
+
+        <label for="{{ form.my_field.id_for_label }}">...</label>{{ my_field }}
+
+    By default, this will be the field's name prefixed by ``id_``
+    ("``id_my_field``" for the example above). You may modify the ID by setting
+    :attr:`~django.forms.Widget.attrs` on the field's widget. For example,
+    declaring a field like this::
+
+        my_field = forms.CharField(widget=forms.TextInput(attrs={'id': 'myFIELD'}))
+
+    and using the template above, would render something like:
+
+    .. code-block:: html
+
+        <label for="myFIELD">...</label><input id="myFIELD" type="text" name="my_field" />
+
+.. attribute:: BoundField.is_hidden
+
+    Returns ``True`` if this :class:`~django.forms.BoundField`'s widget is
+    hidden.
+
+.. attribute:: BoundField.label
 
-.. code-block:: html+django
+    The :attr:`~django.forms.Field.label` of the field. This is used in
+    :meth:`~BoundField.label_tag`.
 
-    <label for="{{ form.my_field.id_for_label }}">...</label>{{ my_field }}
+.. attribute:: BoundField.name
 
-By default, this will be the field's name prefixed by ``id_``
-("``id_my_field``" for the example above). You may modify the ID by setting
-:attr:`~django.forms.Widget.attrs` on the field's widget. For example,
-declaring a field like this::
+    The name of this field in the form::
 
-    my_field = forms.CharField(widget=forms.TextInput(attrs={'id': 'myFIELD'}))
+        >>> f = ContactForm()
+        >>> print(f['subject'].name)
+        subject
+        >>> print(f['message'].name)
+        message
 
-and using the template above, would render something like:
+Methods of ``BoundField``
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. method:: BoundField.as_hidden(attrs=None, **kwargs)
+
+    Returns a string of HTML for representing this as an ``<input type="hidden">``.
+
+    ``**kwargs`` are passed to :meth:`~django.forms.BoundField.as_widget`.
+
+    This is primarily used internally. You should use a
+    :class:`~django.forms.HiddenInput` instead.
+
+.. method:: BoundField.as_widget(widget=None, attrs=None, only_initial=False)
+
+    Renders the field by rendering the passed widget, adding any HTML
+    attributes passed as ``attrs``.  If no widget is specified, then the
+    field's default widget will be used.
+    ``only_initial`` is used by Django internals and should not be set
```

Comment:
If this is meant as a separate paragraph it needs a blank line above it. If not, flow it to the previous line.